### DISCUSSION TITLE
How do I get Adgurad home DNS to actually work?

### DISCUSSION THREAD

**Discussion by DCCXVIII** [AdguardTeam/AdGuardHome#6963] [Q&A]
Don't know what I'm doing wrong. I've setup Adguard home on a raspberry pi 5. Logged into the Adguard home firmware page. Set all that up. Set a static IP for the Raspberry Pi. Set my router (Synology) to point to the Raspberry Pi (both the preferred and the alternate). Got traffic coming through the Adguard control panel. But noticed that the only traffic hitting it was from my Synology router. All the devices on my LAN all seem to be completely ignoring the new DNS settings and blowing right passed Adguard. 

What do?

**Comment by Dynamic5912** [comment]
Set the LAN DNS to the IP address of the RPi on the Synology.
